tmt/checks/dmesg.py

```diff
@@ -202,10 +205,16 @@ def after_test(
             environment: Optional[tmt.utils.Environment] = None,
             logger: tmt.log.Logger) -> list[CheckResult]:
         if not invocation.guest.facts.has_capability(GuestCapability.SYSLOG_ACTION_READ_ALL):
-            return [CheckResult(name='dmesg', result=ResultOutcome.SKIP)]
+            return [CheckResult(
+                name='dmesg',
+                result=ResultOutcome.SKIP,
+                note=['fetching of buffer content was not allowed'])]
 
         if not invocation.is_guest_healthy:
-            return [CheckResult(name='dmesg', result=ResultOutcome.SKIP)]
+            return [CheckResult(
+                name='dmesg',
+                result=ResultOutcome.SKIP,
+                note=['guest was not healthy'])]
```

Comment:
Also this one, I had to look at code to understand what does it mean the guest is not healthy, currently that means that restart or hard reboot was requested. With this note, it would be hard to guess what this actually means.

Reply:
Addressed, the text is now much richer.